examples/simple_example.rs

```diff
@@ -41,14 +41,35 @@ extern crate libc;
 use traceme::Tracer;
 use libc::getpid;
 
+use traceme::backends::DummyTracer;
+#[cfg(perf_pt)]
+use traceme::backends::PerfPTTracer;
+
+
+/// Instantiate a tracer suitable for the current platform.
+fn init_tracer() -> Box<Tracer> {
```

Comment:
I think this function name is misleading. It doesn't initialise a tracer: it makes a new one. So maybe just call it `tracer`?

Reply:
Sure, but it might disappear, see the comment above about tests:
https://github.com/softdevteam/traceme/pull/19#discussion_r160356429